go.mod

```diff
@@ -1,56 +1,56 @@
 module github.com/snyk/go-application-framework
 
-go 1.18
+go 1.21
```

Comment:
@PeterSchafer this should be ok, now that CLI is using a higher go version, right?

Reply:
We are on go 1.21 now